twyla/service/examples/consumer.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+import os
+
+import twyla.service.events as events
+
+
+async def listener(event_name):
```

Comment:
Will this kind of abstraction be done in each service using the event bus service, or will this kind of `listener` eventually be a part of the event bus service?

Reply:
Basically `twyla.service` is similar to `twyla.shared` in that it is supposed to be used as library. It is more of a framework than shared though.